test/core/derivatives/implementation/backpack.py

```diff
@@ -38,7 +38,8 @@ def jac_t_mat_prod(self, mat):  # noqa: D102
 
     def weight_jac_t_mat_prod(self, mat, sum_batch, subsampling=None):  # noqa: D102
```

Comment:
Can we remove all `param_jac_t` functions in the BackPACK and autograd implementation for the tests?

Reply:
Yes, I tried it and it worked out.